release_notes/v2_vs_v3.adoc

```diff
@@ -16,6 +16,10 @@ Many of the same terms are used, and the same functions are performed, but the t
 
 This topic discusses these differences in detail, in an effort to help OpenShift users in the transition from OpenShift v2 to OpenShift v3.
 
+ifdef::openshift-dedicated[]
+OpenShift Dedicated uses the same code base as OpenShift Enterprise 3. OpenShift Dedicated is hosted on the public cloud and managed by Red Hat. An OpenShift Enterprise 3 subscription entitles you to host and manage the software on your infrastructure of choice.
+endif::[]
```

Comment:
Changed to:

"OpenShift Dedicated 3 follows the numbering of the product's major version, and uses the same code base as OpenShift Enterprise 3."

I don't think the sentence explaining what "v2/3" is too relevant, as it's pretty self explanatory. I did, however, change the first instance of "OpenShift v2 and v3" to OpenShift version 2 and OpenShift version 3" just in case. Like how you spell out abbreviations the first time, then use them then on.

Kept it as a note box as well.